dspace-spring-rest/src/test/java/org/dspace/app/rest/MetadataSchemaRestRepositoryIT.java

```diff
@@ -7,50 +7,218 @@
+            .build();
 
         getClient().perform(get("/api/core/metadataschemas"))
-                   .andExpect(status().isOk())
-                   .andExpect(content().contentType(contentType))
-                   .andExpect(jsonPath("$._embedded.metadataschemas", Matchers.hasItem(
-                       MetadataschemaMatcher.matchEntry()
-                   )))
-                   .andExpect(jsonPath("$._links.self.href", Matchers.containsString("/api/core/metadataschemas")))
-                   .andExpect(jsonPath("$.page.size", is(20)));
+                .andExpect(status().isOk())
+                .andExpect(content().contentType(contentType))
+                .andExpect(jsonPath("$._embedded.metadataschemas", Matchers.hasItem(
+                        matchEntry()
+                )))
+                .andExpect(jsonPath("$._links.self.href", Matchers.containsString("/api/core/metadataschemas")))
+                .andExpect(jsonPath("$.page.size", is(20)));
     }
 
-
     @Test
     public void findOne() throws Exception {
 
         context.turnOffAuthorisationSystem();
+        MetadataSchema metadataSchema = MetadataSchemaBuilder.createMetadataSchema(context, "ATest", "ANamespace")
+                .build();
+
+        getClient().perform(get("/api/core/metadataschemas/" + metadataSchema.getID()))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$", is(
+                        matchEntry(metadataSchema)
+                )));
+    }
+
+    @Test
+    public void createSuccess() throws Exception {
+        context.turnOffAuthorisationSystem();
+
         MetadataSchema metadataSchema = MetadataSchemaBuilder.createMetadataSchema(context, "ATest", "ANamespace")
                                                              .build();
 
+
+
+        MetadataSchemaRest metadataSchemaRest = metadataSchemaConverter.fromModel(metadataSchema);
+        metadataSchemaRest.setPrefix(TEST_NAME);
+        metadataSchemaRest.setNamespace(TEST_NAMESPACE);
+
+        String authToken = getAuthToken(admin.getEmail(), password);
+        AtomicReference<Integer> idRef = new AtomicReference<>();
+
+
+        getClient(authToken)
+                .perform(post("/api/core/metadataschemas")
+                        .content(new ObjectMapper().writeValueAsBytes(metadataSchemaRest))
+                        .contentType(contentType))
+                .andExpect(status().isCreated())
+                .andDo(result -> idRef.set(read(result.getResponse().getContentAsString(), "$.id")));
+
+        getClient().perform(get("/api/core/metadataschemas/" + idRef.get()))
+                   .andExpect(status().isOk())
+                   .andExpect(jsonPath("$", MetadataschemaMatcher.matchEntry(TEST_NAME, TEST_NAMESPACE)));
+    }
+
+    @Test
+    public void createUnauthauthorizedTest()
+            throws Exception {
+        context.turnOffAuthorisationSystem();
```

Comment:
I believe the Auth system should remain on in this method.  We aren't creating test data in this method.

Also, this method name has a misspelling. It should be `createUnauthorizedTest()`